jobs/azWebAppDeployJob.yml

```diff
@@ -192,6 +192,21 @@ jobs:
               pipeline: ${{ parameters.CIPipelineId }}
               runVersion: latestFromBranch
               runBranch: ${{ parameters.deployBranch }}
+          
+        - task: AzureRmWebAppDeployment@4
+          name: azureDeployment
+          displayName: Deploy to ${{ parameters.slot }} Slot
+          inputs:
+            azureSubscription: ${{ parameters.subscription }}
+            WebAppName: ${{ parameters.appName }}
+            deployToSlotOrASE: true
+            ResourceGroupName: ${{ parameters.resourceGroup }}
+            SlotName: ${{ parameters.slot }}
+            packageForLinux:  $(Pipeline.Workspace)/App/App.zip
```

Comment:
should this be a variable to make sure it is consistent through out the template as I have seen it in few places?

Reply:
Variable'd now